main.py

```
@@ -1,170 +1,7 @@
-import lmstudio as lms
-from lmstudio import ToolFunctionDef
-import time
-import pick
-import requests
-import asyncio
-from crawl4ai import AsyncWebCrawler
-from crawl4ai.async_configs import BrowserConfig, CrawlerRunConfig
+from model_loader import load_model, does_chat_fit_in_context
```

Ruff (0.11.9)

`model_loader.load_model` imported but unused

Remove unused import

(F401)